DigitalLearningSolutions.Data/Helpers/DisplayStringHelper.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace DigitalLearningSolutions.Web.Helpers
+﻿namespace DigitalLearningSolutions.Data.Helpers

```

Comment:
I've moved the DisplayStringHelper over to the Data project. Originally, it was in the Web project to discourage the generation of display strings until there were necessary in view models and views. But this is indeed a case where we want to use the PRN display string in the data project, so I've just moved the whole thing over. Hence there are a lot of file changes to modify used namespaces..

Reply:
I'm kind of tempted to say you should split off the PRN functionality into its own helper, and leave the rest of the file in the web project.

This is what I did with the display names, splitting off NameQueryHelper which does the "Lastname, Firstname".